images/common/utils.sh

```
@@ -86,7 +86,7 @@ function wait_nginx_services {
 	set +e
 	while :; do
 		wget -qS ${DASHBOARD_INTERNAL}/admin/login/ 2>&1 | grep -q "200 OK"
-		if [[ $? = "0" ]]; then
+		if [ $? = "0" ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)